PROGRESSION(298518@main?): [iOS Debug] imported/w3c/web-platform-t ests/digital-credentials/al low-attribute-with-create.h ttps.html is failing

martadarbinyan opened the pull request:
#### 278449c45cfd92b44d8337d8d3c46632f2f7e86a
<pre>
PROGRESSION(298518@main?): [iOS Debug] imported/w3c/web-platform-t ests/digital-credentials/al low-attribute-with-create.h ttps.html is failing
<a href="https://bugs.webkit.org/show_bug.cgi?id=297364">https://bugs.webkit.org/show_bug.cgi?id=297364</a>
<a href="https://rdar.apple.com/158257364">rdar://158257364</a>

Reviewed by Ryan Haddad.

Rebaselining the test that has a progression.

* LayoutTests/platform/ios/imported/w3c/web-platform-tests/digital-credentials/allow-attribute-with-create.https-expected.txt: Added.
* LayoutTests/platform/ipad/imported/w3c/web-platform-tests/digital-credentials/allow-attribute-with-create.https-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/298970@main">https://commits.webkit.org/298970@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/df5797650e0e2aa5105555aa3bce9cf51ba98bc7

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/117173 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/36820 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/27424 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/123240 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/69168 "Built successfully") | ⏳ 🛠 ios-apple 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/37532 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/45425 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/88940 "Found 1 new test failure: compositing/patterns/direct-pattern-compositing-position.html (failure)") | [💥 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/43625 "An unexpected error occured. Recent messages:Printed configuration") | ⏳ 🛠 mac-apple 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/120106 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/29899 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/105076 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/69428 "Passed tests") | | ⏳ 🛠 vision-apple 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/28956 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/23189 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/66892 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/99296 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/23376 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/126376 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/44059 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/33114 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/97613 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/44414 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/101312 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/97398 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/42753 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/20683 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/40433 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/18727 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/43934 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/43401 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/46762 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/45111 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/5cb938426e337f9a0f6ad16b34bde85aced95517. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/49553

ryanhaddad reviewed (approved):
r=me

martadarbinyan committed:
PROGRESSION(298518@main?): [iOS Debug] imported/w3c/web-platform-t ests/digital-credentials/al low-attribute-with-create.h ttps.html is failing
https://bugs.webkit.org/show_bug.cgi?id=297364
rdar://158257364

Reviewed by Ryan Haddad.

Rebaselining the test that has a progression.

* LayoutTests/platform/ios/imported/w3c/web-platform-tests/digital-credentials/allow-attribute-with-create.https-expected.txt: Added.
* LayoutTests/platform/ipad/imported/w3c/web-platform-tests/digital-credentials/allow-attribute-with-create.https-expected.txt: Added.

Canonical link: https://commits.webkit.org/298970@main
